src/components/new-suggestion/progress-bar/progress-bar.tsx

```diff
@@ -17,25 +18,39 @@ function showSongInformationError() {
   toast.warn("You need to fill in all the required fields before continuing")
 }
 
+function showInstrumentError() {
+  toast.warn("You need to add at least one instrument before continuing")
+}
+
 const ProgressBar = () => {
   const dispatch: AppDispatch = useDispatch()
   const activeArea = useSelector((state: AppState) => state.newSuggestion.activeArea)
+  const newSuggestion = useSelector((state: AppState) => state.newSuggestion)
   const { watch } = useFormContext<InputsSongInformation>()
 
   function colorArea(area: string) {
     return area === activeArea ? "text-moon-300" : "text-zinc-300"
   }
 
   function goToInstruments() {
+    submitSongInformationForm()
     if (isSongInformationInvalid(watch)) showSongInformationError()
     else dispatch(setActiveArea(Area.Instruments))
-    submitSongInformationForm()
   }
 
   function goToReview() {
-    if (isSongInformationInvalid(watch)) showSongInformationError()
-    else dispatch(setActiveArea(Area.Review))
     submitSongInformationForm()
+    if (isSongInformationInvalid(watch)) {
+      showSongInformationError()
+      return
+    }
+
+    if (isInstrumentSuggestionInvalid(newSuggestion.suggestion.instruments)) {
+      showInstrumentError()
+      return
+    }
```

Comment:
Maybe for the flow set the active area to instrument? That way the user can immediately see where they need to add an instrument.